modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -709,6 +737,12 @@ public CompletableFuture<Replica> startReplica(
 
         return newRaftClientFut.thenComposeAsync(raftClient -> {
             if (!busyLock.enterBusy()) {
+                try {
+                    raftManager.stopRaftNodes(replicaGrpId);
```

Comment:
Looks strange, why we stop raft node on start? If without this code everything is still broken, I think we should to fix the real reason of race

Reply:
The implementation is changed in way to collect raft-nodes and stopping them manually on `ReplicaManager#stopAsync` if there are unreachable nodes from replicas.